src/shared/components/CountdownTimer/components/TimerList/TimerList.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { TimerItem } from './components/TimerItem';
+import { periods } from '../../api/constants/periods';
+import './TimerList.scss';
+
+interface TimerListProps {
+  values: number[];
+}
+
+export const TimerList = ({ values }: TimerListProps) => (
```

Comment:
what are values? Use more convenient name

Reply:
changed to timerValues